container-core/src/main/java/com/yahoo/metrics/ContainerMetrics.java

```diff
@@ -40,7 +44,10 @@ public enum ContainerMetrics {
     JETTY_THREADPOOL_TOTAL_THREADS("jdisc.http.jetty.threadpool.thread.total", Unit.THREAD,
             "Current number of threads"),
     JETTY_THREADPOOL_QUEUE_SIZE("jdisc.http.jetty.threadpool.queue.size", Unit.THREAD,
-            "Current size of the job queue");
+            "Current size of the job queue"),
+
+    SERVER_NUM_CONNECTIONS("serverNumConnections", Unit.RESPONSE,
```

Comment:
@gjoranv : Do we need a new unit for connections, or should we add something generic? Not sure what this will be used for

Reply:
Yes, I suggest to add new units for `connection` and `request`. Units are, at least currently, only for documentation, so no need to be restrictive.

Reply:
Added. Thanks